src/java/com/palantir/cassandra/objects/Dropwizard4Cluster.java

```diff
@@ -0,0 +1,32 @@
+package com.palantir.cassandra.objects;
+
+import com.codahale.metrics.jmx.JmxReporter;
+import com.datastax.driver.core.Cluster;
+
+public class Dropwizard4Cluster extends Cluster {
```

Comment:
nit: would add a comment linking to https://docs.datastax.com/en/developer/java-driver/3.6/manual/metrics/index.html#metrics-4-compatibility for posterity